po/de.po

```diff
@@ -15972,9 +15972,8 @@ msgid ""
 "change direction"
 msgstr ""
 "Klick und ziehen fügt Punkt hinzu.\n"
-"Scrollen, um die Größe zu ändern. \n"
-"Umschalt+scrollen, um die Stärke zu ändern. \n"
-"Strg+scrollen, um die Richtung zu ändern."
+"Scrollen, um die Größe zu ändern. - Umschalt+scrollen, um die Stärke zu "
```

Comment:
I'm not sure, it certainly won't fit on a single line.

Reply:
![darktable-10936](https://user-images.githubusercontent.com/6246352/151308236-c41a0279-c913-430f-8d69-a988e95c18e0.png)

It certainly would fit.

That's what it looks like at 1920x1080 (the minimum resolution at which comfortable work in darktable is still possible) with system font size.

It fits even at 1440x900 with a reasonable font size change to make it not so big at that resolution.

Don't forget that top banner (the output area of hinter messages) is not limited by the edge of left panel, so it can accommodate for quite long strings!

Reply:
Ok, I'm really not fond of editing .po directly. But seems safe here. Can you resolve the conflict on de.po? TIA.

Reply:
Conflict resolved.